app/config/config.yml

```diff
@@ -136,11 +136,12 @@ cmf_sonata_phpcr_admin_integration:
         seo: ~
         menu: ~
         core: ~
-        content: ~
+        content:
```

Comment:
had the configuration under `content:` on my first approach and moved to a global one then. So this change is wrong